src/test/task.test.js

```diff
@@ -29,3 +33,61 @@ describe('Remove Task', () => {
     expect(tasks.length).toBe(2);
   });
 });
+
+describe('Edit Task', () => {
+  it('Edit task description', () => {
+    const update = {
+      description: 'edit do some coding task',
+      completed: false,
+      index: 1,
+    };
+    expect(editTask(update, tasks)).toEqual([
+      {
+        description: 'edit do some coding task',
+        completed: false,
+        index: 1,
+      },
+      {
+        description: 'learn another language',
+        completed: false,
+        index: 3,
+      },
+    ]);
+  });
+
+  it('Check edited task description', () => {
+    expect(tasks[0].description).toBe('edit do some coding task');
+  });
+
+  it('Check task as completed', () => {
+    const task = {
+      description: 'go to school',
+      completed: false,
+      index: 3,
+    };
+    expect(completed(3, task, true)).toEqual({
+      description: 'go to school',
+      completed: true,
+      index: 3,
+    });
+  });
+});
+
+describe('Clear completed', () => {
+  const removeCompleted = [
+    { description: 'do some excercise', completed: false, index: 1 },
+    { description: 'cooks some food', completed: true, index: 2 },
+    { description: 'go hiking', completed: true, index: 3 },
+  ];
+
+  it('Clear all completed tasks', () => {
+    expect(clearCompleted(removeCompleted)).toEqual([
+      { description: 'do some excercise', completed: false, index: 1 },
+    ]);
+  });
+
+  it('check length of array after removing completed tasks', () => {
+    const checkCompleted = clearCompleted(removeCompleted);
+    expect(checkCompleted).toHaveLength(1);
+  });
```

Comment:
- [ ] You did excellent work :clap:. I think you can also add a test for the function for updating an item's index value upon item deletion or dragging if applicable.  :pray:

Reply:
@henatan99 thank you for the review, but the drag and drop function was optional and I did not implement it in my project. also when an item is deleted the index do not update, rather what happens is when adding a new task the function checks the existing tasks for the max index number and adds one to the max index number to create a new index

Reply:
Hi, @yaseerokino, I said the dragging only if applicable. But at least the app should update the item's index when an item is deleted. For example, if you have items with index: 1, index: 2, and index: 3, and if you delete an item with index: 2, then an item with index: 3, should be updated to index: 2.  Therefore , you need to add a test if the indexes are updated after the list is changed. :pray:

Reply:
i have made the required changes